Frameworks/Foundation/NSDecimalNumber.mm

```diff
@@ -312,34 +350,98 @@ - (NSDecimalNumber*)decimalNumberByRaisingToPower:(NSUInteger)power {
  @Status Stub
  @Notes
 */
-- (NSDecimalNumber*)decimalNumberByMultiplyingByPowerOf10:(short)power {
+- (instancetype)initWithString:(NSString*)numericString locale:(id)locale {
     UNIMPLEMENTED();
     return StubReturn();
 }
 
 /**
- @Status Stub
- @Notes
+ @Status Interoperable
+*/
+- (NSDecimal)decimalValue {
+    // Read-Only
+    NSDecimal decimal = { 0 };
```

Comment:
-  f {a=0 b={c=0.000000000 d=0x00dbfb20 {0, 0, 0, 0} } e=0 ...} Foo
  a 0 int
-  b {c=0.000000000 d=0x00dbfb20 {0, 0, 0, 0} } Foo::B
  c 0.000000000 float
-  d 0x00dbfb20 {0, 0, 0, 0} __int64[4]
  [0] 0 __int64
  [1] 0 __int64
  [2] 0 __int64
  [3] 0 __int64
  e 0 int
+  bar {baz=0 } Bar
-  f2 {a=0 b={c=0.000000000 d=0x00dbfaf0 {61919066037682176, 11289035728869430, 8392977107035880220, 87311050130587648} } ...} Foo
  a 0 int
-  b {c=0.000000000 d=0x00dbfaf0 {61919066037682176, 11289035728869430, 8392977107035880220, 87311050130587648} } Foo::B
  c 0.000000000 float
-  d 0x00dbfaf0 {61919066037682176, 11289035728869430, 8392977107035880220, 87311050130587648} __int64[4]
  [0] 61919066037682176 __int64
  [1] 11289035728869430 __int64
  [2] 8392977107035880220 __int64
  [3] 87311050130587648 __int64
  e 0 int
+  bar {baz=0 } Bar


on x86 Release

---
In reply to: [81210095](https://github.com/Microsoft/WinObjC/pull/1032#discussion_r81210095) [](ancestors = 81210095,80795216,80787871)